docs/changelog.rst

```diff
@@ -1,8 +1,37 @@
 =========
 Changelog
 =========
-
 * :release:`0.100.3-rc5 <2019-05-08>`
```

Comment:
Finally a comment on the too many bugs. In the past bugs between RCs were not added in the changelog as they are not supposed to be user facing bugs as users are not supposed to use RCs. If we do the changelog blows up like it does right here and imo loses its meaning.

Reply:
I removed all bug fixes which were fixed in between releases.